app/src/components/TimeLine.js

```diff
@@ -27,26 +27,40 @@ function TimeLine({ timeRows, isAuthenticated, dayStart, totalHours }) {
       {getTimes().map((time, index) => {
         return <TimeNotch key={`TimeNotch-${index}`} {...{ time }} />;
       })}
-      <TimeIndicator {...{ dayStart }} />
+
+      {getTimes().map((time, index) => {
+        if (index === 0) {
+          return <TimeIndicator key={"current-time"} {...{ dayStart }} />;
+        }
+        return <TimeNotch key={`TimeNotch-${index}-0`} time={null} />;
+      })}
     </div>
   );
 }
 
 function TimeNotch({ time }) {
   return (
     <div
-      key={time.timeString}
-      className="timeNotch"
-      style={{ gridColumn: "1", gridRow: time.timeLineLocation }}
+      key={time === null ? "blank" : time.timeString}
+      className={time === null ? "timeNotch null" : "timeNotch"}
+      style={{
+        gridColumn: time === null ? "2" : "1",
+        gridRow: time === null ? "auto" : time.timeLineLocation,
+      }}
     >
-      {time.timeString}
+      {time ? (
+        <span>{time.timeString}</span>
+      ) : (
+        <span className="nullTime">space</span>
+      )}
```

Comment:
There are a bunch of ternary checks to see whether time is null here.  You might consider doing two separate components, a `NullTimeNotch` component and a regular `TimeNotch` one:

```
function NullTimeNotch() {
  return (
    <div
      key={"blank"}
      className={"timeNotch null"}
      style={{
        gridColumn: "2",
        gridRow: "auto",
      }}
    >
      <span className="nullTime">space</span>
    </div>
  );
}

function TimeNotch({ time }) {
  if(time == null) return <NullTimeNotch />;

  return (
    <div
      key={time.timeString}
      className={"timeNotch"}
      style={{
        gridColumn: "1",
        gridRow: time.timeLineLocation,
      }}
    >
      <span>{time.timeString}</span>
    </div>
  );
}
```